scripts/sct_extract_metric.py

```diff
@@ -46,6 +45,35 @@ def __init__(self):
         self.perlevel = None
 
 
+class _ListLabelsAction(argparse.Action):
+    """This class makes it possible to call the flag '-list-labels' without the need to input the required '-i'."""
+    def __init__(self,
+                 option_strings,
+                 dest=argparse.SUPPRESS,
+                 default=argparse.SUPPRESS,
+                 help=None):
+        super(_ListLabelsAction, self).__init__(
+            option_strings=option_strings,
+            dest=dest,
+            default=default,
+            nargs=0,
+            help=help)
```

Comment:
Is this `__init__` necessary? I was able to get rid of it by adding `nargs=0,` to the `-list-label` definition.

Reply:
Implemented in 7cbe901, thank you for the suggestion!

Reply:
And regarding the motivation for putting this `__init__` in the first place: I applied the solution proposed in this [stackoverflow thread](https://stackoverflow.com/questions/34352405/python-argparse-help-like-option) without taking the time to properly analyze it (it worked so I just went ahead with it due to lack of time)